File: public/javaPrograms/conditionalsAndLoops/Two.java

package javaPrograms.conditionalsAndLoops;
import java.util.Scanner;
public class Two {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        System.out.print("Enter a no. :");
        int n = s.nextInt();
        int mul = 1;
        int add  = 0;
        while(n>0){
            int rem = n%10;
             mul = mul*rem;
             add = add+rem;
            n/=10;
        }
        System.out.println(
            mul-add
        );
        
    }
}

File: src/programs/arrays/two.js

import '../../styles/programs.css'
import '../../styles/topics.css'
import ConditionalsAndLoops from '../../topics/ConditionalsAndLoops'
import { useState,useEffect } from 'react'
import ConditionalsAndLoopsThree from './three'
import Arrays from '../../topics/Arrays'
import ArraysThree from './three'
const ArraysTwo = ({qn,backPage}) =>{
    const q3 = "Input a number and print all the factors of that number (use loops)."
    const back = (page) =>{
        backPage(page)
    }
        const url = window.location.href

        const [file,setFile] = useState("Loading Java FIle...")
    
        useEffect(()=>{
        async function fetchFile() {
            try{
        const res = await fetch(`${url}javaPrograms/conditionalsAndLoops/Two.java`)
   
        const cont = await res.text();
        
        setFile(cont)
        }
        catch(err){
        console.log(err);
        }

    }
     
    fetchFile() 
  },[])
    return(<div className="program-body">
        <div className='topics-logo'>2.{qn}</div>
        <div className='program-content'>

          
            <div className='program-hint program'>
                <pre>  {file}</pre>
         

            </div>
            <div className='program-btns'>
            <div onClick={() => back(<Arrays/>)} className='program-btn'>Go To Map</div>
            <div onClick={() => back(<ArraysThree qn ={q3} backPage={backPage}/>)} className='program-btn'>Next</div>
                </div>
            
        </div>
    
    </div>)
}
export default ArraysTwo